### PR TITLE
Abbey

### PR DESCRIPTION
**Pull request by abbeycallow**

### CODE REVIEW

**Comment by abbeycallow** [comment]
This is the bank account app.

File: src/com/company/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+        do {
+
+            System.out.println("Press 1 to check balance.");
+
+            System.out.println("Press 2 to deposit funds.");
+
+            System.out.println("Press 3 to withdraw funds.");
+
+            System.out.println("Press 4 to exit menu. ");
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: I would have this say something like "exit application" so the user knows the app will close

File: src/com/company/BankAccount.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                case 3:
+                    //withdraw(amount);
+                    withdraw(balance);
+                    break;
+
+                case 4:
+                    System.out.println("exit menu.");
+                    break;
+
+
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: Delete extra whitespace

File: src/com/company/BankAccount.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    public void deposit(int amount) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("How much would you like to deposit?");
+        amount = Integer.parseInt(scanner.nextLine());
+
+
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: delete extra white space

File: src/com/company/BankAccount.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            balance += amount;
+
+            System.out.println(amount + " Has been deposited.");
+            System.out.println("your new balance is : " + balance);
+
+
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: delete extra white space

File: src/com/company/BankAccount.java [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        if (amount > balance) {
+            System.out.println("your don't have enough money.");
+        } else {
+
+            balance = amount - amount;
+            //balance = balance - amount;
+            balance = amount;
+
+
+
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: delete extra white space

File: src/com/company/BankAccount.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+                case 1:
+                    System.out.println("Your balance is : " + balance );
+                    break;
+
+                case 2:
+                    deposit(amount);
+                    break;
+
+                case 3:
+                    //withdraw(amount);
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: delete this comment

File: src/com/company/BankAccount.java [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+                    withdraw(balance);
+                    break;
+
+                case 4:
+                    System.out.println("exit menu.");
```
Comment (mbsnapit) [inline src/com/company/BankAccount.java]: Have message say something like "Thanks for you using this app" or "Goodbye"

**Review by mbsnapit — CHANGES_REQUESTED** [review]

**Review by mbsnapit — APPROVED** [review]
Looks good! +1

**Pull request merged**